files/avian-influenza/Incidence_2.csv

```diff
@@ -0,0 +1,53 @@
+MidTime,Count
```

Comment:
can you please rename (and update links) to `incidence.csv` (note lower case) - I don't think that there is a version 1 of this file.

Reply:
Updated, there is no version 1 - it was just called Incidence_v2.csv in the file supplied